src/assets/stylesheets/base/_forms.scss

```diff
@@ -0,0 +1,29 @@
+form {
+  display: flex;
+  flex-direction: column;
+  width: rem(320px);
+  height: rem(270px);
```

Comment:
@gutakk Do we really need to set this, we already set the container width?

Reply:
You're right. Fixed in 272c1662fb02cd83f73cdd6e01d246f0adfa2117